vendor/runner/cli.js

```diff
@@ -0,0 +1,101 @@
+// Usage:
+//
+// Run all vendors:
+//   npm run test:vendor
+//
+// Run some vendors:
+//   npm run test:vendor -- --only=vendor1,vendor2,vendor3
+//
+// Infer the current PR from the Travis environment, and look for bracketed,
+// space-separated vendor names in the pull request title. If none are found,
+// do not run any tests. For example:
+// Pull request title: [travis sonar] Support user token authentication
+//   npm run test:vendor -- --pr
+// is equivalent to
+//   npm run test:vendor -- --only=travis,sonar
+
+'use strict';
+
+const request = require('request');
+const minimist = require('minimist');
+const difference = require('lodash.difference');
+const serverHelpers = require('../../test/in-process-server-helpers');
+const Runner = require('./runner');
+
+const getTitle = (repoSlug, pullRequest) => new Promise((resolve, reject) => {
+  const options = {
+    uri: `https://api.github.com/repos/${repoSlug}/pulls/${pullRequest}`,
+    json: true,
+    headers: { 'User-Agent': 'badges/shields' },
+  };
+  request(options, (err, res, json) => {
```

Comment:
Maybe use a promise wrapper for `request`? See https://github.com/request/request#promises--asyncawait. Alternatively, use a `fetch` polyfill rather than using `request`?